lib/jira.js

```diff
@@ -207,7 +207,7 @@ var JiraApi = exports.JiraApi = function(protocol, host, port, username, passwor
 
         var options = {
             rejectUnauthorized: this.strictSSL,
-            uri: this.makeUri('/issue/' + issueNumber),
+            uri: this.makeUri('/issue/' + issueNumber + '?expand=renderedFields'),
```

Comment:
Should we make this the default option? I think there should be a way to specify this value.